installer/upgrader/Migrations/V5/generalLangString.yaml

```
@@ -480,7 +480,7 @@ langStrings:
     value: 'Reject'
     unitId: reject
   -
-    value: '{amount,plural, =0{No Records{action}} one{1 Record {action}} other{# Records {action}}}'
+    value: '{count,plural, =0{No Records{action}} one{1 Record {action}} other{# Records {action}}}'
```

```suggestion
    value: '{count,plural, =0{No Records Found} one{1 Record Found} other{# Records Found}}'
```
It seems better to remove action. Otherwise it will be again text concatination

---

So this is used for 'selected' as well and not just 'found'  
Do you want me to keep separate strings for found and select then?

---

yeah having separate strings seem to make sense i'll do as you suggested

---

changed